var/spack/repos/builtin/packages/py-pyhdf/package.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPyhdf(PythonPackage):
+    """pyhdf is a python wrapper around the NCSA HDF version 4 library.
+    The SD (Scientific Dataset), VS (Vdata) and V (Vgroup) APIs are
+    currently implemented. NetCDF files can also be read and modified."""
+
+    homepage = "https://github.com/fhs/pyhdf"
+    pypi = "pyhdf/pyhdf-0.10.4.tar.gz"
+    git = "https://github.com/fhs/pyhdf.git"
+
+    maintainers("climbfuji")
+
+    license("MIT")
+
+    version("master", branch="master")
+    version("0.10.4", sha256="ea09b2bdafc9be0f7f43d72ff122d8efbde61881f4da3a659b33be5e29215f93")
+
+    # Python versions
+    depends_on("python@3.2:", type=("build", "run"))
```

Comment:
Should adjust this to whatever minimum Python spack requires? Or drop entirely since it's always going to be fine?

Reply:
IMO It's sometimes nice to be explicit but others would disagree.  At present, the `python` package deprecates `3.7.x` and has no earlier versions.  I'll leave that decision to you.

Reply:
Thanks! I'll leave it as is then - fewer changes is always good ;-)

Reply:
:thinking: I don't recall seeing a python version listed in any setup-related files for either the branch or version.